From: win7-support-bot <win7@cromite.org>
Date: Sun, 26 Jul 2026 00:00:00 +0000
Subject: WIN7: components-crash / com

Files:
 components/crash/core/app/fallback_crash_handling_win.cc

License: GPL-2.0-or-later - https://spdx.org/licenses/GPL-2.0-or-later.html
---
diff --git a/components/crash/core/app/fallback_crash_handling_win.cc b/components/crash/core/app/fallback_crash_handling_win.cc
index fad310cd6bf7c..194af822969fc 100644
--- a/components/crash/core/app/fallback_crash_handling_win.cc
+++ b/components/crash/core/app/fallback_crash_handling_win.cc
@@ -65,7 +65,6 @@ bool SetupFallbackCrashHandling(const base::CommandLine& command_line) {
       new FallbackCrashHandlerLauncher());
 
   if (!fallback_launcher->Initialize(base_command_line, database_path)) {
-    NOTREACHED();
   }
 
   // This is necessary because chrome_elf stubs out the
